### PR TITLE
Feature/callback entry transformer

### PR DESCRIPTION
**Pull request by pambuk**
<h2>Change Log</h2>
<div id="change-log">
  <h4>Added</h4>
  <ul id="added">
    <li>Added Callback Entry transformer</li>
  </ul> 
  <h4>Fixed</h4>  
  <ul id="fixed">
    
  </ul>
  <h4>Changed</h4>
  <ul id="changed">
    
  </ul>  
  <h4>Removed</h4>
  <ul id="removed">
    
  </ul>
  <h4>Deprecated</h4>
  <ul id="deprecated">
    
  </ul>  
  <h4>Security</h4> 
  <ul id="security">
    
  </ul>     
</div>
<hr/>

<h2>Description</h2>

For those cases where we have to rename a lot of entries but all renaming is done according to a simple pattern, like replacing dashes - with underscores _

### CODE REVIEW

File: src/Flow/ETL/Transformer/CallbackEntryTransformer.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public function __construct(callable ...$callables)
+    {
+        $this->callables = $callables;
+    }
+
+    /**
+     * @psalm-suppress InvalidArgument
+     * @psalm-suppress MixedInferredReturnType
+     * @psalm-suppress MixedAssignment
+     * @psalm-suppress MixedReturnStatement
```
Comment (norberttech) [inline src/Flow/ETL/Transformer/CallbackEntryTransformer.php:30]: I think in this case all those suppressions are redundant and they all refer to the $callables that you can type using docblocks

Comment (pambuk) [reply src/Flow/ETL/Transformer/CallbackEntryTransformer.php]: I removed `Mixed...` suppersions by adding proper docblock for `$callable` (remaining one relates to `Flow\ETL\Rows::map expects pure-callable`)

Comment (norberttech) [reply src/Flow/ETL/Transformer/CallbackEntryTransformer.php]: https://gist.github.com/norberttech/f8144bb5b4d19e1051e9da7312399104 this is how you can remove everything

Comment (pambuk) [reply src/Flow/ETL/Transformer/CallbackEntryTransformer.php]: Fixed, thanks :)

**Comment by norberttech** [comment]
🙌 could you please put also example into README before I merge this?

**Comment by pambuk** [comment]
Example added.

**Comment by norberttech** [comment]
This is great, thanks!

**Pull request merged**